install/cluster_synced.go

```diff
@@ -112,7 +112,7 @@ func (c *SyncedCluster) Stop() {
 		}
 		defer session.Close()
 
-		cmd := `pkill -9 "cockroach|java|mongo|kv|ycsb" || true ;
+		cmd := `pkill -9 "^(cockroach|java|mongo|kv|ycsb)$" || true ;
```

Comment:
Killing `java` probably isn't a good idea on localhost. I bet we can remove `cockroach`, `java` and `mongo` from this list and rely on the `lsof`-based killing below.

Reply:
I was thinking the same, killing random processes is a bad idea in general when the `local` cluster is used.

Reply:
Btw, this originated from using `roachperf` on non-ephemeral clusters with cockroach started by other means than `roachperf`. It is very likely overkill.